gaffer-as-a-service/gaas-eureka-server/src/main/java/uk/gov/gchq/gaffer/eurekaserver/EurekaServerApplication.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package demo.eurekaserver;
+package uk.gov.gchq.gaffer.eurekaserver;
```

Comment:
This might be better in `uk.gov.gchq.gaffer.gaas.demo.eurekaserver`?

Reply:
I have added gaas as part of the package however don't think we should include the word demo. This eureka server is expected to run alongside rest in normal deployments.

Reply:
Sounds good, didn't know if this still wanted to be a demo